include/nbl/asset/utils/CCPUMeshPackerV1.h

```diff
@@ -304,8 +262,9 @@ typename CCPUMeshPackerV1<MDIStructType>::ReservedAllocationMeshBuffers CCPUMesh
 
 		return invalidReservedAllocationMeshBuffers;
 	}
-	
-	if (m_vtxBuffAlctrResSpc)
+
+	bool arePerVtxAttribsEnabled = alctrTraits::get_total_size(m_vtxBuffAlctr) == 0 ? false : true;
+	if (arePerVtxAttribsEnabled)
 	{
 		vtxAllocAddr = m_vtxBuffAlctr.alloc_addr(vtxCnt * m_vtxSize, 1u);
 		if (vtxAllocAddr == INVALID_ADDRESS)
```

Comment:
have a look at `SRAIIBasedExiter`, you could wrap up all the 4 failure cases into it

P.S. I might actually do it myself

Reply:
if it works, it works, I'm leaving it alone